BridgeSDK/BridgeAPI/SBBBridgeNetworkManager.m

```diff
@@ -74,11 +74,12 @@ - (instancetype)initWithAuthManager:(id<SBBAuthManagerInternalProtocol>)authMana
     return self;
 }
 
-- (NSURLSessionDataTask *) doDataTask: (NSString*) method
-                            URLString: (NSString*)URLString
-                              headers: (NSDictionary *)headers
-                           parameters:(NSDictionary *)parameters
-                           completion:(SBBNetworkManagerCompletionBlock)completion
+- (NSURLSessionTask *)doDataTask:(NSString *)method
```

Comment:
Is this intended as an internal method only? (ie. should the old version be deprecated?)